Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -141,6 +141,18 @@ public void SerializesAndDeserializesCustomPropertyNames()
             Check(TestObjects.fmRadio);
         }
 
+        [TestMethod]
+        public void SerializesAndDeserializesWithCustomObjectSerializer()
+        {
+            Check(TestObjects.Rover);
+        }
+        
+        [TestMethod]
+        public void SerializesAndDeserializesChildObjectWithCustomObjectSerializer()
+        {
+            Check(TestObjects.JohnDoe);
```

Comment:
Do we need both tests? I think this test would be sufficient, given that it's testing simple fields as well as complex ones?

Reply:
Removed redundant test.